### PR TITLE
fix: Svg - type error ReactComponent not defined 

### PR DESCRIPTION
**Pull request by sravan-s**

### CODE REVIEW

**Comment by sravan-s** [comment]
Screen Shot 2023-06-27 at 10 26 35 AM

File: src/customs.d.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+declare module '*.svg' {
+  import React = require('react');
+  export const ReactComponent: React.FC<React.SVGProps<SVGSVGElement>>;
+  const src: string;
+  export default src;
+}
```
Comment (liamcho) [inline src/customs.d.ts:6]: why we need this?

**Review by liamcho — APPROVED** [review]
룩스굿투미

File: src/customs.d.ts (hunk 1 above)
Comment (sravan-s) [reply src/customs.d.ts:6]: Its tricking Ts think it will get a ReactComponent from SVG file

**Pull request merged**